src/main/java/com/napier/sem/CapitalCitiesReport.java

```diff
@@ -0,0 +1,21 @@
+package com.napier.sem;
+
+import java.util.List;
+
+public class CapitalCitiesReport implements ReportCapitalCities{
+
+    @Override
+    public void capitalCitiesWorld() {
+        CapitalCitiesDAO capitalcitiesDao = new CapitalCitiesDAOImpl();
+        List<CapitalCities> capitalcities = capitalcitiesDao.getAllCapitalCities();
+        for (CapitalCities c:capitalcities){
+            System.out.println(c.toString());
+        }
+    }
+
+    @Override
+    public void generateReport() {
+        capitalCitiesWorld();
```

Comment:
keeping the same naming convention as the other report should be fine here, so the method name can be `largestToSmallest`

Reply:
and maybe the `interface Report` could be reused